model/host/host.go

```diff
@@ -1019,6 +1019,14 @@ func FindAllRunningParentsByContainerPool(poolId string) ([]Host, error) {
 	return Find(query)
 }
 
+// CountUninitializedParents returns the number of initializing parent host intent documents
+func CountUninitializedParents() (int, error) {
+	return db.Count(Collection, bson.M{
+		HasContainersKey: true,
+		StatusKey:        evergreen.HostUninitialized,
```

Comment:
To answer your question - the status you should check here is any host in any of the evergreen.UpHostStatus values. Every host in that status should be treated as an active parent

Reply:
thanks, done.